cfgov/jinja2/v1/test-fixture/index.html

```diff
@@ -0,0 +1,118 @@
+{% set atomic_type = request.GET.get( 'atomic' ) %}
+{% set atomic_type = 'header' if atomic_type == None else atomic_type %}
+
+<!DOCTYPE html>
+
+<!--[if IE 8]>         <html lang="en" class="no-js lt-ie10 lt-ie9"> <![endif]-->
+<!--[if IE 9]>         <html lang="en" class="no-js lt-ie10"> <![endif]-->
+<!--[if gt IE 8]><!--> <html lang="en" class="no-js"> <!--<![endif]-->
+
+<head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# {% block og_article_prefix %}{% endblock %}">
+
+<!--
+    ===========
+    GLOBAL META
+    ===========
+-->
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta charset="utf-8">
+    {% block meta_viewport %}
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% endblock meta_viewport %}
+
+<!--
```

Comment:
The meta info could be packaged up for reuse on both v1 and non-v1 instead of duplicating